---
title: '&#39;Abfangen&#39; Block wird niemals erreicht; &lt;Ausnahme&gt; wird oben in der gleichen behandelt &#39;versuchen&#39; Anweisung'
ms.date: 07/20/2015
f1_keywords:
- bc42031
- vbc42031
helpviewer_keywords:
- BC42031
ms.assetid: 7d15597c-5988-42ea-a853-63cbf78faaf3
ms.openlocfilehash: f84eced8f36e404886ae1b435bbb3ba1beffebbd
ms.sourcegitcommit: 3c1c3ba79895335ff3737934e39372555ca7d6d0
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 09/05/2018
ms.locfileid: "43786959"
---
# <a name="39catch39-block-never-reached-ltexceptiongt-handled-above-in-the-same-39try39-statement"></a>&#39;Abfangen&#39; Block wird niemals erreicht; &lt;Ausnahme&gt; wird oben in der gleichen behandelt &#39;versuchen&#39; Anweisung
Ein `Catch` -Block im Code kann nicht erreicht werden, weil er in einem vorhergehenden `Try` -Block behandelt wird.  
  
 Standardmäßig ist diese Meldung eine Warnung. Weitere Informationen zum Ausblenden von Warnungen oder zum Behandeln von Warnungen als Fehler finden Sie unter [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic)  
  
 **Fehler-ID:** BC42031  
  
## <a name="to-correct-this-error"></a>So beheben Sie diesen Fehler  
  
-   Entfernen Sie die redundante Anweisung.  
  
## <a name="see-also"></a>Siehe auch  
 [Vorgehensweise: Auffangen einer Ausnahme in Visual Basic](https://msdn.microsoft.com/library/f3063c89-d2bf-49b1-91b5-b87edfb18b95)  
 [Vorgehensweise: Testen von Code mit einem... Catch-Block in Visual Basic](https://msdn.microsoft.com/library/8368e205-ed73-4185-a247-af84fb4fafa9)  
 [Vorgehensweise: Filtern von Fehlern in einem CatchBlock in Visual Basic](https://msdn.microsoft.com/library/85964d0a-56e7-4301-a96e-5eaea23b7b9b)  
 [Exemplarische Vorgehensweise: Strukturierte Ausnahmebehandlung (Visual Basic)](https://msdn.microsoft.com/library/440da655-4b32-490b-8b16-bfe46f41fa76)  
 [Try...Catch...Finally-Anweisung](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
